src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200226135950_AddReleaseIdToFootnotes.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
```

Comment:
Please can you alter file useful-scripts\sql\FootnoteData.sql 

There are 22 footnote inserts near the top of the file (id, content) which need to become (id, content, releaseId).

Here is an example:
```
INSERT INTO Footnote (Id, Content)
VALUES (@footnote_id_1, 'State-funded primary schools include all primary academies, including free schools.');
```

This is just because this file isn't currently included in the InitialCreate_Custom migrations that run so it will be run manually after this new migration in the new production environment when we get it.

It could be included...but it would need to be tested. 😄

Reply:
I think this change is going to break datafactory because it won't have a mapping in its configuration to the new release id destination column. The configuration lives in the ARM template.